cmd/rekor_monitor/main.go

```diff
@@ -163,7 +162,6 @@ func main() {
 		}
```

Comment:
Can we remove the default of 0? Scrapping the entire log is hard to do without careful rate limiting on the client side. If no start index is provided and no checkpoint is provided, then just throw an error.